yelp.sql

```diff
@@ -0,0 +1,74 @@
+-- const data = require ()
+
+
+DROP DATABASE IF EXISTS yelp;
+
+CREATE DATABASE yelp;
+
+USE yelp;
+
+CREATE TABLE restaurants
+(
+  id INTEGER AUTO_INCREMENT NOT NULL,
+  names varchar (150) NOT NULL,
+  cuisine varchar(150) NOT NULL,
+  PRIMARY KEY (ID)
+);
+
+CREATE TABLE pictures
+(
+  id int  AUTO_INCREMENT NOT NULL,
+  caption varchar(255) NOT NULL,
+  images  LONGBLOB NOT NULL,
```

Comment:
maybe change data type from LONGBLOB to TEXT since it will be a url

Reply:
Thanks, I will change it